function/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+)
+
+func plus(a int, b int) int {
+	return a + b
+}
+func plusPlus(a, b, c int) int {
+	return a + b + c
+}
+func mul(x, y, z int) int {
+	return x * y * z
+}
```

Comment:
All is good, but always keep your returned parameters with name, and I hope you know the purpose of camelCase and TitleCase in Golang Function Naming?????